src/main/java/apoc/export/util/ExportFormat.java

```diff
@@ -36,7 +37,7 @@ public static final ExportFormat fromString(String format) {
                 }
             }
         }
-        return NEO4J_SHELL;
+        return CYPHER;
```

Comment:
Oh, I thought this format stuff was unreleased. Given it is, then you're right - leave that change for your 3.2 release.